.travis.yml

```diff
@@ -1,43 +1,24 @@
 language: python
 sudo: true
-python:
-  - "3.5"
-addons:
-  apt:
-    packages:
-    - g++
-    - gfortran
-    - libblas-dev
-    - liblapack-dev
-    - libopenblas-dev
-    - python3-dev
-    - enchant
-    - redis-server
-
+env:
+  - DOCKER_COMPOSE_VERSION=1.17.1
+services:
+  - docker
 before_install:
-  - npm install -g eslint
-  - sudo sed -e 's/^bind.*/bind 127.0.0.1/' /etc/redis/redis.conf > redis.conf
-  - sudo mv redis.conf /etc/redis
-  - sudo service redis-server start
-  - echo PING | nc localhost 6379
```

Comment:
What's going on here?  Looks like this needs a comment.

Reply:
All of this is being handled in the docker-compose. Docker compose brings up the redis and all of the needed dependencies.

Reply:
I think it's showing the wrong side of the diff.  See, the code you added removes docker-compose and re-downloads it.  Why?

Reply:
aha, docker compose is not installed by default, it needs to be installed: https://docs.travis-ci.com/user/docker/#using-docker-compose

Reply:
I removed that part completely, it's not needed.